### PR TITLE
compress svg file

### PR DESCRIPTION
**Pull request by rickyseezy**

### CODE REVIEW

**Review by snroki — APPROVED** [review]

File: _includes/twitter_logo.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-</g>
+<svg width="22pt" height="22pt" version="1.1" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```
Comment (ch3ric) [inline _includes/twitter_logo.html:2]: what tool did you use to optimize these svg?
for example, on master this svg file is 1,6kB big. Whereas on your branch it's 1,1kB, but when I use `svgomg` on my side I get 699B.

File: _includes/rss_logo.html [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-  <path d="M11.8871 20.217C11.8871 21.1928 12.6855 21.9911 13.6613 21.9911C14.6371 21.9911 15.4355 21.1928 15.4355 20.217C15.4355 12.6944 9.31452 6.55566 1.77419 6.55566C0.798387 6.55566 0 7.35405 0 8.32986C0 9.30566 0.798387 10.1041 1.77419 10.1041C7.3629 10.1041 11.8871 14.646 11.8871 20.217Z" fill="white"/>
-  <path d="M20.2256 21.9912C21.2015 21.9912 21.9998 21.1928 21.9998 20.217C21.9998 9.07504 12.9337 0.00891113 1.79177 0.00891113C0.815965 0.00891113 0.0175781 0.807298 0.0175781 1.7831C0.0175781 2.75891 0.815965 3.5573 1.79177 3.5573C10.9821 3.5573 18.4515 11.0267 18.4515 20.217C18.4515 21.1928 19.2321 21.9912 20.2256 21.9912Z" fill="white"/>
-  </g>
-  <defs>
-  <clipPath id="clip0">
-  <rect width="26" height="26" fill="white"/>
-  </clipPath>
-  </defs>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="30" height="30" fill="none" viewBox="0 0 26 26" xmlns="http://www.w3.org/2000/svg">
```
Comment (ch3ric) [inline _includes/rss_logo.html:2]: I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox

File: _includes/facebook_logo.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-<g id="surface1">
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(23.137255%,34.901961%,59.607843%);fill-opacity:1;" d="M 20.71875 21.925781 C 21.386719 21.925781 21.925781 21.386719 21.925781 20.722656 L 21.925781 1.28125 C 21.925781 0.613281 21.386719 0.0742188 20.71875 0.0742188 L 1.28125 0.0742188 C 0.613281 0.0742188 0.0742188 0.613281 0.0742188 1.28125 L 0.0742188 20.722656 C 0.0742188 21.386719 0.613281 21.925781 1.28125 21.925781 Z M 20.71875 21.925781 "/>
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 15.152344 21.925781 L 15.152344 13.464844 L 17.992188 13.464844 L 18.417969 10.164062 L 15.152344 10.164062 L 15.152344 8.058594 C 15.152344 7.105469 15.417969 6.453125 16.785156 6.453125 L 18.53125 6.453125 L 18.53125 3.503906 C 18.230469 3.464844 17.195312 3.375 15.988281 3.375 C 13.46875 3.375 11.746094 4.910156 11.746094 7.734375 L 11.746094 10.164062 L 8.898438 10.164062 L 8.898438 13.464844 L 11.746094 13.464844 L 11.746094 21.925781 Z M 15.152344 21.925781 "/>
-</g>
+<svg width="22pt" height="22pt" version="1.1" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```
Comment (ch3ric) [inline _includes/facebook_logo.html:2]: I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox

File: _includes/twitter_logo.html (hunk 1 above)
Comment (ch3ric) [inline _includes/twitter_logo.html:2]: I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox

File: _includes/twitter_logo.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+    <g>
+        <rect width="22" height="22" fill="#4fabf1"/>
```
Comment (ch3ric) [inline _includes/twitter_logo.html:4]: I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox

**Review by ch3ric — COMMENTED** [review]
could you please also optimize these other svg files at the same time:
- _includes/wttj_logo.html
- _includes/close_search.html
- _includes/linkedin_logo.html
- _includes/logo.html
- _includes/search_mobile_icon.html
- _includes/locale_logo.html
- _includes/eleven_logo.html

File: _includes/linkedin_logo.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 18.675781 11.667969 C 18.675781 9.734375 17.109375 8.171875 15.179688 8.171875 L 14.625 8.171875 C 13.570312 8.171875 12.636719 8.699219 12.074219 9.503906 C 12.015625 9.589844 11.957031 9.683594 11.90625 9.777344 C 11.886719 9.773438 11.878906 9.773438 11.878906 9.773438 L 11.878906 8.644531 C 11.878906 8.527344 11.78125 8.433594 11.667969 8.433594 L 8.972656 8.433594 C 8.855469 8.433594 8.761719 8.527344 8.761719 8.644531 L 8.761719 18.445312 C 8.761719 18.5625 8.855469 18.65625 8.972656 18.65625 L 11.726562 18.65625 C 11.839844 18.65625 11.9375 18.5625 11.9375 18.445312 L 11.9375 12.78125 C 11.9375 11.804688 12.714844 10.992188 13.695312 10.980469 C 14.195312 10.972656 14.648438 11.175781 14.976562 11.5 C 15.296875 11.824219 15.496094 12.269531 15.496094 12.761719 L 15.496094 18.449219 C 15.496094 18.5625 15.589844 18.65625 15.707031 18.65625 L 18.464844 18.660156 C 18.582031 18.660156 18.675781 18.5625 18.675781 18.449219 Z M 18.675781 11.667969 "/>
-</g>
+<svg width="22pt" height="22pt" version="1.1" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```
Comment (ch3ric) [inline _includes/linkedin_logo.html:2]: I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox

**Pull request closed without merging**